cli/server.go

```diff
@@ -982,6 +989,23 @@ func (r *RootCmd) Server(newAPI func(context.Context, *coderd.Options) (*coderd.
 			options.WorkspaceUsageTracker = tracker
 			defer tracker.Close()
 
+			// Manage notifications.
+			var notificationsManager *notifications.Manager
```

Comment:
nit: could also initialize this as a `NoopManager` and overwrite if experiment not enabled

Reply:
I didn't want to implement the full contract of `Manager` as an interface; the key bit is just the enqueueing.